libvisual/CMakeLists.txt

```diff
@@ -185,14 +193,29 @@ ENDIF()
 
 # Git revision
 FIND_PACKAGE(Git)
-CONFIGURE_FILE(
-  "${PROJECT_SOURCE_DIR}/update-version.sh.in"
-  "${PROJECT_BINARY_DIR}/update-version.sh"
-  @ONLY
-)
-ADD_CUSTOM_TARGET(update-version ALL
-  COMMAND /bin/sh ${PROJECT_BINARY_DIR}/update-version.sh
-)
+
+EXECUTE_PROCESS(COMMAND ${GIT_EXECUTABLE} describe --always --dirty
+  OUTPUT_VARIABLE LV_REVISION OUTPUT_STRIP_TRAILING_WHITESPACE)
```

Comment:
Because this command is no longer in `ADD_CUSTOM_TARGET` but run unconditionally, this will get users building from tarball stderr output like this (I just tried):
```
fatal: not a git repository (or any parent up to mount point /)
Stopping at filesystem boundary (GIT_DISCOVERY_ACROSS_FILESYSTEM not set).
```
It would be nice to not do that, to maybe not call Git in that case, at all. Maybe we can call Git if the version file is missing?

Reply:
Forgot to add `IF(GIT_FOUND)` to limit that. Should the default stay as unknown?